spec/fixtures/_complete_rizzo.json

```diff
@@ -81,6 +81,51 @@
       "hostname": "jumpbox.acme.com",
       "ip": "172.16.100.26",
       "boot_timeout": 300
+    },
+    {
+      "name": "windows",
+      "hostname": "windows.acme.com",
+      "windows": true,
+      "ip": "172.16.100.27",
+      "forwarded_ports": [
+        {
+          "guest": "5985",
+          "host": "5985"
+        }
+      ],
+      "gui": true,
+      "bootstrap_repo_path": "/Users/jeff/projects/acme/bootstrap_windows",
+      "bootstrap_script_path": "bootstrap_puppet4.ps1",
+      "bootstrap_script_args": "",
+      "bootstrap_guest_path": "C:\\tmp\\bootstrap_puppet4",
+      "box": "win2016",
+      "box_url": "https://artifactory.acme.net/artifactory/infra-vagrant-local/windows2016.box",
+      "box_download_checksum": "123456781038992207555513504e37258c29e2e9",
+      "box_download_checksum_type": "sha1",
+      "update_packages": "false",
+      "shutdown": "false"
+    },
+    {
+      "name": "windows2",
+      "hostname": "windows2.acme.com",
+      "windows": true,
+      "ip": "172.16.100.28",
+      "forwarded_ports": [
+        {
+          "guest": "15985",
+          "host": "15985"
+        }
+      ],
+      "bootstrap_repo_path": "/Users/jeff/projects/acme/bootstrap_windows",
+      "bootstrap_script_path": "bootstrap_puppet4.ps1",
+      "bootstrap_script_args": "arg1 arg2",
+      "bootstrap_guest_path": "C:\\tmp\\bootstrap_puppet4",
+      "box": "win2016",
+      "box_url": "https://artifactory.acme.net/artifactory/infra-vagrant-local/windows2016.box",
+      "box_download_checksum": "123456781038992207555513504e37258c29e2e9",
+      "box_download_checksum_type": "sha1",
+      "update_packages": "false",
+      "shutdown": "false"
```

Comment:
`update_packages` and `shutdown` have to be string `"false"` instead of boolean `false` to get this to work. 

Added the following to `lib/rzo/app/templates/Vagrantfile.erb` to aid in debugging and found that these two options were always showing as `true`, even if set to `false` here. This is odd as other options from defaults such as `box` work as expected.

```
shutdown = x<%= nc['shutdown'] %>x
update_packages = x<%= nc['update_packages'] %>x
```

Reply:
The `false` becoming `true` issue appears to be a "feature" or some quirk with deep_merge gem.  https://github.com/danielsdeleo/deep_merge/issues/23.  I think the summary is that deep_merge handles values of different classes in a way that isn't entirely intuitive and `true` is a different class from `false`.

I've not yet found a way to force a `false` option to override the `true` default.

Reply:
Does it work if we convert booleans to strings and then back to booleans after the merge?

Reply:
I think so but haven't tested that approach yet.  The last comment on the issue I linked seems to indicate that's a valid workaround.  I'll test that out and see how viable an option that is.